src/app/api/schemas.py

```diff
@@ -95,21 +95,26 @@ class EventOut(EventIn, _CreatedAt):
 
 
 class DeviceIn(BaseModel):
-    name: str = Field(..., min_length=3, max_length=50)
+    name: str = Field(..., min_length=3, max_length=50, example="pyronearEngine51")
     owner_id: int = Field(..., gt=0)
     specs: str = Field(..., min_length=3, max_length=100)
```

Comment:
Perhaps some examples on ̀owner_id`and `specs` while you're at it?

Reply:
thanks, I have indeed forgotten "specs" !

For all ids, I was wondering if it is usefull to generate some because, there is a defaut id 0 when doc is generated.  I will add some examples then